### ISSUE TITLE
Commodities with no data are displaying in commodity drop-down for production in Explore Data

### ISSUE THREAD

**Issue by mcharg** [ONRR/nrrd#1506]
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/1ac6d483-be5d-45c7-92b7-ae56bb44af39)

**Comment by mcharg** [comment]
@mojobnichols Looks like maybe there's 2020 data in the database for this commodity, but we're not displaying 2020 in Explore Data.
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/3457d8ea-7bdb-4750-ad42-6816bb5a2649)

**Comment by mcharg** [comment]
Another example.
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/6d764017-6c5e-4ae7-9bb6-3983b22ce8d2)

**Comment by mcharg** [comment]
Borrow sand & gravel doesn't display in the query tool.

**Comment by Maroyafaied** [comment]
@mcharg Yes, you can assign me and Mojo to it. I'll look at it first to see if it's still an issue.

**Comment by mojobnichols** [comment]
@mcharg  I see Coal - fines (circuit) for 2020. I don't see Coal-bitimus raw for 2020 for locations I do see them for other years BUT only for Native American - which doesn't show up on map. 


![image.png](https://images.zenhubusercontent.com/5d1a3e771449416ed8985985/47498595-bbb5-4773-8b68-0a45a4cacbb2)

Also Borrow sand & gravel does show up  - Again Native American so you get blank map.   

I at first was puzzled because the query on page (dynamic query) doesn't have either so the fix is to use that, but then this am I got to thinking this is explore data not find the data you know about. So in the case of Borrow sand & gravel I was able to explore and find it in 2006 and 2007 and Native American.  So I'm thinking the dynamic menu based on period and year would be bad as you couldn't find Borrow ... ETC. 

Maybe a quick chat during sync up tomorrow.  Or I can put in query for menu that is based on year which doesn't have items not in that year.  

And @Maroyafaied  since your on the issue.
